md/polling.md

```diff
@@ -42,7 +42,14 @@ There are REST endpoints to manage execution of the polling on `/api/polling/` s
 
 ## Troubleshooting
 
+### Polling errors
+
 When a polling execution fails, it can be seen in error in the polling page [http://localhost:8082/](http://localhost:8082/).
 
 More informations whould be available in output logs on the backend, either in the Standard output or in the log file `logs/bonita-ici.log`
  
+### Deleted cases
+
+When a started case is already polled in ICI storage and then deleted in Bonita platform, this case is still present 
+in ICI storage and will still be displayed in Operation Management Living app, but this overview can't be displayed 
+anymore since data is deleted from Bonita database .   
```

Comment:
maybe we can say, that in elasticsearch should be cleared? (and later that we will have a "clean" button maybe)

Reply:
that could be the first step, but cleaning elastic will loose model & target duration. it's a kind of re-install from scratch (that it did during demo)